### ISSUE TITLE
Check relaxation of fn/fnt, gn/gnt

### ISSUE THREAD

**Issue by dslmeinte** [ehn-dcc-development/eu-dcc-schema#138]
The following text was prepared in the release notes for 1.3.1, but no corresponding change was made to the schema.

Rules around fn/fnt and gn/gnt have been relaxed. Now only one set must be provided. This supports cultures who only have given names and cultures who only have family names.

**Comment by ryanbnl** [comment]
We'll need to make a new release for this:

```
    "person_name": {
      "description": "Person name: Surname(s), forename(s) - in that order",
      "required": [
        "fnt"
      ]
```

Should  be:

```
    "person_name": {
      "description": "Person name: Surname(s), forename(s) - in that order"
```

Or even better - if we can express in JSON Schema that either ```fnt``` or ```gnt``` are required.

**Comment by dslmeinte** [comment]
I think it should be expressed in the JSON Schema, and the documentation needs to be aligned with the guidelines.

**Comment by ryanbnl** [comment]
Agreed - if it can be expressed, then it should be.

The If..Then..Else construct should allow it to be expressed:

https://json-schema.org/understanding-json-schema/reference/conditionals.html

**Comment by lari-kannisto** [comment]
> We'll need to make a new release for this:
> 
```
    "person_name": {
      "description": "Person name: Surname(s), forename(s) - in that order",
      "required": [
        "fnt"
      ]
 ```
> 
> Should be:
> 
```
    "person_name": {
      "description": "Person name: Surname(s), forename(s) - in that order"
```
> 
> Or even better - if we can express in JSON Schema that either `fnt` or `gnt` are required.

Hi!

I think the following should work:

```json
    "person_name": {
      "description": "Person name: Surname(s), forename(s) - in that order",
      "anyOf":[
        {"required": [
        "fnt"
      ]},
        {"required": [
        "gnt"
      ]}
      ],
      "type": "object",
      "properties": {
        "fn": {
           ...
       }
```

**Comment by dslmeinte** [comment]
(issue is targeted by PR #139)

**Comment by ryanbnl** [comment]
(renamed the branch so the PR changed to #141

**Comment by dslmeinte** [comment]
Solved by PR #141
